README

```diff
@@ -1,17 +1,21 @@
 Term::EditLine
 ==============
 
-Term::EditLine is a object oriented perl interface to the NetBSD editline
-library. To use the module you will have to install this library. Read
+Term::EditLine is an object oriented perl interface to the NetBSD editline
+library. To use the module you will have to install this library first. Read
 the pod documentation for more information on this module.
 
 INSTALLATION
 
+Install libedit, for instance with this command in Ubuntu:
```

Comment:
nit: prefer to refer to Debian here rather than Ubuntu as the latter is derived from the former.  please reword to something like "in Debian" or "in Debian based distributions (including Ubuntu)".

Please also add the instructions for RedHat based distros:

```
sudo yum install libedit-devel
```